dm/openapi/spec/dm.yaml

```
@@ -1617,3 +1765,35 @@ components:
       required:
         - "total"
         - "data"
+
+    BatchImportTaskConfigRequest:
```

In dmctl it's called as `config import`. I don't think we should change the name here.
```suggestion
    ImportTaskConfigRequest:
```